### ISSUE TITLE
Crash using Windowed Mode ASI

### ISSUE THREAD

**Issue by VIRUXE** [carlos-menezes/sa-asi-loader#2]
![image](https://github.com/carlos-menezes/sa-asi-loader/assets/1616657/045b76fb-3e4b-4e30-80b4-b445f3998b85)

Up until now, I've only been testing using one asi file which was for SAMPCAC.

Tried using Windowed Mode and it crashes. Last time I used it was last year.

https://github.com/ThirteenAG/III.VC.SA.WindowedMode

**Comment by carlos-menezes** [comment]
That's strange, because I use the Windowed Mode mod. Does it happen when running `gta_sa.exe`?

**Comment by VIRUXE** [comment]
It does actually.

**Comment by carlos-menezes** [comment]
I assume it doesn't happen if you use SAMPCAC?

**Comment by VIRUXE** [comment]
You're right. Silly me, didn't even think it was this anti-cheat. First time I've used it.
